feature-1/one-morefile.txt

```diff
@@ -0,0 +1,3 @@
+This file was also added in feature-1 branch
```

Comment:
It's really good that you have decided to add this line of code.

Reply:
Thanks for this comment, Vishon1